indexd/blueprint.py

```diff
@@ -66,7 +66,7 @@ def get_record(record):
                 ret = blueprint.alias_driver.get(record)
             except AliasNoRecordFound:
                 if not blueprint.dist or "no_dist" in flask.request.args:
-                    raise
+                    raise IndexNoRecordFound("no record found")
```

Comment:
You went this far... you might as well print the the record to an error log for the thing that could not be found and in the exception.